apps/web/src/components/molecules/List/List.tsx

```diff
@@ -0,0 +1,59 @@
+import clsx from 'clsx'
```

Comment:
I know this has been brought from frf which is fine but probably worth bringing the [unit tests](https://github.com/PA-NIHR-CRN/frf-web/blob/779e884d63d463ee1bb8b06b29f126f681ca5e24/src/components/List/List.spec.tsx) too to keep the coverage high